src/mlpack/tests/split_data_test.cpp

```diff
@@ -891,6 +891,9 @@ TEST_CASE("SplitMatDataShuffleWithLabelsAndWeights", "[SplitDataTest]")
     REQUIRE(found[c]);
 
   // Make sure we can find each column of the labels.
+  labels.print("labels");
+  trainLabels.print("trainLabels");
+  testLabels.print("testLabels");
```

Comment:
why do we want to print them ?

Reply:
Thanks!  This was some debugging that accidentally snuck in.  I removed it now.